Dockerfile.ci

```diff
@@ -167,6 +167,19 @@ RUN mkdir -pv /usr/share/man/man1 \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
 
+#Install MS SQL Drivers
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && curl https://packages.microsoft.com/config/debian/9/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+    && apt-get update -yqq \
+    && apt-get upgrade -yqq \
+    && ACCEPT_EULA=Y apt-get -yqq install -y --no-install-recommends \
+      gcc \
+      unixodbc-dev  \
+      g++ \
+      msodbcsql17 \
+      mssql-tools \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Note to self: look at how big this makes the layer (and if we need to keep _all_ of these in the resulting image?